stellargraph/layer/gcn_lstm.py

```diff
@@ -129,8 +129,7 @@ def build(self, input_shapes):
             input_shapes (list of int): shapes of the layer's inputs (node features and adjacency matrix)
```

Comment:
Should this be something like
```suggestion
            input_shapes (list of int): shapes of the layer's inputs (batch dimension, number of nodes, number of time steps)
```
?

Reply:
Ah, I think this was actually referring to the inputs, so I'll switch it to

```
input_shapes (list of int): shapes of the layer's inputs (the batches of node features)
```